js/components/Drives.tsx

```diff
@@ -71,27 +73,31 @@ export const Drives = ({ cpu, io, sectors, enhanced, ready }: DrivesProps) => {
     });
 
     const [smartPort, setSmartPort] = useState<SmartPort>();
+    const [unknownStorage, setUnknownStorage] = useState<UnknownStorage>();
```

Comment:
Consider moving these up next to the `useState<Disk2>` line and having all of the `data1`, `data2`, lines follow them all.